lib/AST/Decl.cpp

```diff
@@ -7832,6 +7833,14 @@ VarDecl *VarDecl::getLazyStorageProperty() const {
       {});
 }
 
+bool VarDecl::isGlobalStorage() const {
+  if (!hasStorage()) {
```

Comment:
Should this be `hasStorageOrWrapsStorage()`? I wonder how this interacts with property wrapped static properties...

Reply:
Thanks @xedin good point, and I addressed property wrapped static properties in the most recent commit.